### PR TITLE
TMIB-10

### PR DESCRIPTION
**Pull request by notriddle**
Want to help out with writing TMIB but don't have write access to the repo?
[Go here](https://github.com/bors-ng/bors-ng.github.io/blob/tmib-10/_posts/2017-11-01-tmib-10.md), then click the pencil button. That'll open a pull request for the in-progress edition of TMIB.

These steps can be done immediately after opening the TMIB pull request:

- [x] Fix the "Go here" link at the top of this pull request

- [x] Update the issue number in the `title:` part

- [x] Update the month in the `date:` part

- [x] Update the month in the `summary:`

- [x] Update the URL in the "last month" PR link

These steps need to be done as the month rolls along, though they should probably be proofread at the end:

- [x] Generate the "notable additions" list

- [x] Generate the "new contributors" list

- [x] Find a new featured user

And, these should be done last thing:

- [x] Update the PR count in the PR paragraph

- [x] Create, and link in, next month's PR, so that the published edition links to the next one

- [x] Write a few useful tidbits in the `summary:` part

And publish to third-party syndication services:

- [ElixirStatus](https://elixirstatus.com): copy the summary into their description box, followed by the post's URL
- [dev.to](https://dev.to): @notriddle should approve the draft after the RSS system pulls it in

### CODE REVIEW

**Comment by notriddle** [comment]
@hgrecco @MatthieuDartiailh

Can I feature [pyvisa](https://github.com/pyvisa) as the project-of-the-month?

**Comment by MatthieuDartiailh** [comment]
It is fine by me yes.

**Comment by notriddle** [comment]
Done. Please review my text to make sure everything's accurate.

File: _posts/2017-11-01-tmib-10.md [hunk 1]
```diff
@@ -34,7 +34,17 @@ Want to see your name in this newsletter? Look at [bors starters](https://bors.t
 
 ### Who's using bors?
 
-No featured user this month. 😭
+This month's featured user is [pyvisa][], a Python library for the [Virtual Instrument Software Architecture (VISA)][VISA], not to be confused with the credit card payment processor or the paperwork used to move between countries.
+
+VISA is a standard hardware interface for [Testing and Measurement (T&M) devices][TM]. VISA defines high-level message-passing semantics, as well as protocols for tunneling over common physical connects (some are measurement-specific like GPIB, while others are generic like USB and TCP/IP). It also defines a C API for applications to talk to an implementation of the protocol.
+
+The main [pyvisa][pyvisa-lib] library, as such, is a plugabble frontend with two official backends. One backend uses C FFI, and other backend is a [pure Python implementation][pyvisa-py] of the VISA wire protocol (over USB, TCP/IP, and Serial, though more physical protocol support is in the works). You can also develop your own backend and pyvisa can dynamically load it.
```
Comment (MatthieuDartiailh) [inline _posts/2017-11-01-tmib-10.md]: Not sure how relevant it is to your readers but we use ctypes for the C FFI that in Python should not be confused with the CFFI project (that also does C FFI), adding ctypes in between parenthesis may make sense. Other than this, your description is accurate.

Comment (notriddle) [reply _posts/2017-11-01-tmib-10.md]: Fixed it.

**Pull request merged**